src/libs/ajax/Runtimes.js

```diff
@@ -123,7 +123,17 @@ export const Runtimes = signal => {
     listV2: async (labels = {}) => {
       const res = await fetchLeo(`api/v2/runtimes?${qs.stringify({ saturnAutoCreated: true, ...labels })}`,
         _.mergeAll([authOpts(), appIdentifier, { signal }]))
-      return res.json()
+
+      // [IA-3710] In order to keep the front-end backwards compatible, any Azure tool labels
+      // will be changed to JupyterLab.
+      return res.json().then(runtimeList => {
```

Comment:
we usually prefer to await res.json instead of chaining a` .then`. Also, preferable to reference `toolLabel.JupyterLab` as well to avoid duplicating the string here.

nit: would prefer to use something more descriptive than `element` as a variable name here.

i.e. 

```
const runtimeList = await res.json()
const runtimesWithToolLabelDecorated _.map(runtime =>  {
      if (runtime.labels.tool === 'Azure')  {
        element.labels.tool = 'JupyterLab'
      }
     return element
 }), runtimeList)
return runtimesWithToolLabelDecorated
```